cmd/helm/status.go

```diff
@@ -40,6 +40,7 @@ The status consists of:
 - k8s namespace in which the release lives
 - state of the release (can be: unknown, deployed, uninstalled, superseded, failed, uninstalling, pending-install, pending-upgrade or pending-rollback)
 - list of resources that this release consists of, sorted by kind
+- description of the release (can be completion message or error message, need to enable --show-desc)
```

Comment:
Ordering is slightly incorrect for output:
Move this above `list of resources that this release consists of, sorted by kind`.

Also, can you add `release revision` after `state of release`. It is missing from the list.